kernel/nativecode.ml

```diff
@@ -16,6 +16,8 @@ open Nativeinstr
 open Nativelambda
 open Pre_env
 
+[@@@ocaml.warning "-32-37"]
```

Comment:
37 is unused constructor, probably manipulated where ocaml can't see.
32 is unused value, I'm less sure that this one is spurious.